src/Product/InfoBlock.js

```diff
@@ -0,0 +1,119 @@
+import React, {Component} from 'react';
+import styled from 'styled-components';
+
+import triangle from  '../images/triangle-down.svg';
+
+
+class InfoBlock extends Component {
+  
+  state = {isOpened: false};
+  
+  handleClick = () => {
+    this.setState({isOpened: !this.state.isOpened});
```

Comment:
https://medium.com/@shopsifter/using-a-function-in-setstate-instead-of-an-object-1f5cfd6e55d1